.github/workflows/CI.yml

```diff
@@ -0,0 +1,74 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches: 
+        - master
+    tags:
+      - '*'
+  create:
+    tags:
+      - '*' 
+  pull_request:
+    branches:
+        - master 
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.301    
+    - uses: actions/cache@v2
+      id: nuget-packages
+      with:
+        path: 
+          ~/.nuget/packages
+          ${{ github.workspace }}/VndbSharp/obj/project.assets.json
+        key: ${{ runner.os }}-nuget-${{ hashFiles('**/packages.lock.json') }}
```

Comment:
Change this key to use a new cache name after adding the `/VndbSharp/obbj/project.assets.json` path to the cache